src/http.js

```diff
@@ -25,6 +25,7 @@ export default function http(url, request) {
   if (request.requestInterceptor) {
     request = request.requestInterceptor(request) || request
   }
+  request.credentials = 'same-origin';
```

Comment:
It might be easier to write test if you add this to execute#buildRequest. Then you can test `buildRequest`.